src/components/language-selector.js

```diff
@@ -1,74 +1,49 @@
 import React, { useEffect, useState } from "react";
 import { navigate } from "gatsby";
-import { Grid, Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
-import globe from "../images/logos/globe.png";
+import { Grid, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { useTranslation } from "react-i18next";
+import FeatherIcon from "feather-icons-react";
+import { Link } from ".";
```

Comment:
The imports can be rearranged better. For example, `import { Link } from ".";` can come immediately after you import the components from `react-uswds`.